pyanaconda/core/constants.py

```diff
@@ -137,7 +137,18 @@
 FIRSTBOOT_ENVIRON = "firstboot"
 
 # Tainted hardware
-UNSUPPORTED_HW = 1 << 28
+TAINT_SUPPORT_REMOVED = 27
+TAINT_HARDWARE_UNSUPPORTED = 28
+
+WARNING_SUPPORT_REMOVED = N_(
+    "Support for this hardware has been removed in this major release. Please check the removed "
+    "functionality section of the release notes."
+)
+
+WARNING_HARDWARE_UNSUPPORTED = N_(
+    "This hardware (or a combination thereof) is not supported by Red Hat. For more information "
+    "on supported hardware, please refer to http://www.redhat.com/hardware."
```

Comment:
The site is not useful for Fedora.

Reply:
The kernel taint is not supported on Fedora.